src/Serilog.Sinks.XUnit/Serilog.Sinks.XUnit.csproj

```diff
@@ -9,7 +9,7 @@
     <PackageTags>logging Serilog xUnit output ITestOutputHelper</PackageTags>
     <Copyright>Copyright © 2017 Todd Benning</Copyright>
     <Authors>Todd Benning</Authors>
-    <TargetFrameworks>net45;netstandard1.0</TargetFrameworks>
+    <TargetFrameworks>net452;netstandard1.1</TargetFrameworks>
```

Comment:
I was going to bump the version to 1.1.x, due to the Serilog minor version change, but do you think these require a major version change to 2.0?

Reply:
It depends on how much you can trust the semver of your dependencies. Probably safe to bump the minor but you could bump the major if in doubt.

Reply:
Yeah, it will be a breaking change if anyone is using it in a netstandard1.0 or net45 app, so go ahead and change the `name` field in `azure-pipelines.yml` to 2.0. Everything else looks good.

Reply:
FWIW, I bumped the target frameworks here to match what xunit `2.4.1` [targets](https://www.nuget.org/packages/xunit.core/).